d2renderers/d2animate/d2animate.go

```diff
@@ -74,7 +74,7 @@ func Wrap(rootDiagram *d2target.Diagram, svgs [][]byte, renderOpts d2svg.RenderO
 		return nil, err
 	}
 
-	d2svg.EmbedFonts(buf, diagramHash, svgsStr, rootDiagram.FontFamily)
+	d2svg.EmbedFonts(buf, diagramHash, svgsStr, rootDiagram.FontFamily, rootDiagram.GetUniqueChars())
```

Comment:
you have to create a function like GetNestedUniqueChars() to account for chars that only appear in nested boards, like

https://github.com/terrastruct/d2/blob/a02fc36c358f1170cf6ed696e3378d32b2098fda/d2target/d2target.go#L125